packages/scene-composer/src/components/Tree/tree.scss

```diff
@@ -15,7 +15,6 @@ $active-color: #ec7211;
   background: $active-bg;
 }
 @mixin selected-action {
-  @include selected;
```

Comment:
What's this for?

Reply:
It's a CSS `mixin` that was adding the matching border for the `.action` class. Looks like there was an update from a dependency that modified the sizing so currently there's a nested border showing above & below the action buttons on hover/selected state. Removing the `mixin` to resolve the issue.